WeatherAlert/schema.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "http://json-schema.org/schema#",
   "$schemaVersion": "0.4.1",
+  "validationSchema": "https://github.com/smart-data-models/dataModel.Weather/edit/master/WeatherAlert/schema.json", 
```

Comment:
No, validationSchema is not a key property of the schema.json but a new property that should be defined inside "properties": { }, with the corresponding type and description, where type should be a string, and description should make reference to the corresponding URL in the Model and description about the URL to the corresponding validation schema of the data model and you can put a default value of the link to the validationSchema.